### PR TITLE
UNION DISTINCT

### PR DESCRIPTION
**Pull request by hvub**
This PR documents UNION DISTINCT, cf. https://github.com/neo-technology/neo4j/pull/24723

(builds on #898 )

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-924.surge.sh

Updated pages:
- [UNION](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/clauses/union)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/clauses/union.adoc [hunk 1]
```diff
@@ -92,3 +92,27 @@ The combined result is returned, without duplicates.
 | "Hitchcock"
 |Rows: 3
 |===
+
+Removal of duplicates can also be accomplished by explicitly including `DISTINCT` in the `UNION`.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/clauses/union.adoc:100]:
```suggestion
[role=label--new-5.19]
[[union-distinct]]
=== UNION DISTINCT

Removal of duplicates can also be accomplished by explicitly including `DISTINCT` in the `UNION`.
```

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 2]
```diff
@@ -36,6 +36,17 @@ FINISH
...
 | New xref:clauses/finish.adoc[FINISH] clause, which can be optionally used to define a query that returns no result.
 
+a|
+label:syntax[]
+label:new[]
+[source, cypher, role=noheader]
+----
+  RETURN 1 AS a
+UNION DISTINCT
+  RETURN 1 AS a
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: Is the indenting of the RETURN clauses intentional?

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 3]
```diff
@@ -36,6 +36,17 @@ FINISH
...
+----
+| New optional keyword DISTINCT after xref:clauses/union.adoc[UNION] as the explicit form of a UNION with duplicate removal.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
| The keyword `DISTINCT` can now be added after a xref:clauses/union.adoc#union-distinct[UNION] as the explicit form of a `UNION` with duplicate removal.
```

**Review by JPryce-Aklundh — COMMENTED** [review]
Looks good @hvub - just a few editorial suggestions from me 👍

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-924.surge.sh

Updated pages:
- [UNION](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/clauses/union)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc (hunk 2 above)
Comment (hvub) [reply modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: I have seen people do this. UNION is one level up in the language structure compared to RETURN, hence, it is not an unreasonable thing to do. However, I should have checked how the documentation formats UNION and follow that.

So here we go:

```suggestion
RETURN 1 AS a
UNION DISTINCT
RETURN 1 AS a
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-924.surge.sh

Updated pages:
- [UNION](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/clauses/union)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-924.surge.sh

Updated pages:
- [UNION](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/clauses/union)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Review by JPryce-Aklundh — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-924.surge.sh

Updated pages:
- [UNION](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/clauses/union)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-924.surge.sh

Updated pages:
- [UNION](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/clauses/union)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-924.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**